packages/itwinui-react/src/core/SearchBox/SearchBox.tsx

```diff
@@ -0,0 +1,39 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { InputProps } from '../Input';
+import { useTheme } from '../utils';
+
+export type SearchBoxProps = {
+  /**
+   *
+   */
+  startAdornment?: React.ReactNode | React.ReactNode[];
+  /**
+   *
+   */
+  endAdornment?: React.ReactNode | React.ReactNode[];
+} & InputProps;
+
+/**
+ * // TODO:
+ * Searchbox component.
+ * Used for searches :)
+ * @example
+ * Example usages go here!
+ */
+export const SearchBox = (props: SearchBoxProps) => {
+  const { size, startAdornment, endAdornment, ...rest } = props;
+  useTheme();
+  return (
+    <div className='iui-input-flex-container' data-iui-size={size}>
+      {startAdornment}
+      <input className='iui-invisible-borders' type='search' {...rest} />
+      {endAdornment}
```

Comment:
Sizing and state question:
1. Clone provided elements and add size and disabled attributes 
or 
2. Leave size and state for users to handle

Reply:
cloning has caused problems for our users and is discouraged by official react docs: https://beta.reactjs.org/reference/react/cloneElement

`disabled` makes sense for user to handle. `size` would be nice to handle. is it possible to do it without too much effort? if not, we could leave it for #969

Reply:
Not relevant because we changed the direction of API.